### PR TITLE
DEVEXP-245 Added docs for ArcGIS Feature Service support

### PR DESCRIPTION
**Pull request by rjrudin**
Shifting to a GH Pages approach instead of a single big README file.

### CODE REVIEW

File: docs/feature-service-support.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
```
Comment (rjrudin) [inline docs/feature-service-support.md:1]: This is the main thing to review.

File: docs/feature-service-support.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+The MarkLogic Koop Provider supports multiple operations in the ArcGIS Feature Service API, each of which is detailed
+below.
+
+## Query Layer Operation
+
+The following parameters in the
+[Query Layer operation](https://developers.arcgis.com/rest/services-reference/enterprise/query-feature-service-layer-.htm)
+are supported (these are listed in the order they appear in the ArcGIS documentation):
+
+- `where`
```
Comment (rjrudin) [inline docs/feature-service-support.md:16]: I was thinking - perhaps we describe each of these - but the link above already does that, and I don't know of anything worth adding for any of these that the ArcGIS docs don't already state.

Comment (BillFarber) [reply docs/feature-service-support.md:16]: yeah, there's nothing more to add. Additionally, if anything changes on that page, we'd have to change it here.

**Review by BillFarber — APPROVED** [review]
LGTM

**Pull request merged**